assets/js/app/Instance.js

```diff
@@ -8,8 +8,8 @@
 
   LocusZoom.Instance Class
 
-  An instance is an independent LocusZoom object. Many instances can exist simultaneously
-  on a single page, each having its own data caching, configuration, and state.
+  An Instance is an independent LocusZoom object. Many such LocusZoom objects can exist simultaneously
```

Comment:
We really should think about renaming this soon. Maybe a "Viewer" or "Plot". Having an instance of an Instance is just a bit confusing.